modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To query the cluster resource metrics API.
+* To monitor or manage alerts and silences.
+* To access Alertmanager component.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
+
+|===
+|Name |Description
+
+|`cluster-monitoring-metrics-api` |Users with this role have the ability to query the cluster resource metrics API for core platform monitoring.
+
+|`cluster-monitoring-operator-alert-customization` |Users with this role can monitor `AlertingRule` and `AlertRelabelConfig` resources for core platform monitoring. These permissions are required for the alert customization feature.
+
+|`monitoring-alertmanager-edit` |Users with this role can create and silence alerts for core platform monitoring in the *Administrator* perspective of the {product-title} web console.
+
+|`monitoring-alertmanager-view` |Users with this role have read access to the Alertmanager API for core platform monitoring. They can also view alert silences in the *Developer* perspective of the {product-title} web console.
+
+|`cluster-monitoring-view` |Users with this cluster role can access Alertmanager component for core platform monitoring.
```

Comment:
this one is a bit complicated:
* It's a cluster role which requires a ClusterRoleBinding instead of RoleBinding like the other roles above.
* In practice it's equivalent to `cluster-monitoring-metrics-api` with additional permissions on the `/federate` endpoint of the user-defined Prometheus.